target/linux/ath79/image/generic.mk

```diff
@@ -438,6 +438,13 @@ define Device/jjplus_ja76pf2
 endef
 TARGET_DEVICES += jjplus_ja76pf2
 
+define Device/kuwfi_cpe2g-9531
+  ATH_SOC := qca9531
+  DEVICE_TITLE := KUWFI CPE2G-9531
```

Comment:
Please update to use DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT as introduced in:
https://github.com/openwrt/openwrt/commit/8c3dbaf8b49233e275b2dce4e30dc0a2bf8c86c5

Reply:
fixed